docker/dockerfiles/bedrock_functional_tests

```diff
@@ -0,0 +1,28 @@
+FROM python:2
+
+WORKDIR /app
+
+# Defaults
```

Comment:
What do you think about deduplicating the defaults and keeping in the dockerfile only the docker related defaults?